src/webhooks/pubsub/slackScores.ts

```diff
@@ -266,6 +283,38 @@ export const sendDiscussionMetrics = async (ospo_internal: boolean = false) => {
   scoreBoard += `└${'─'.repeat(
     DISCUSSION_COLUMN_WIDTH + NUM_COMMENTS_COLUMN_WIDTH + NUM_ROW_SPACES
   )}┘`;
+  firstColumnName = 'Most Active Issues this Week';
```

Comment:
Feels like the next 100 or so lines repeat the exact same functionality for each of discussions, issues, and gitHubCommenters. Could we factor our into a standalone helper function?

Reply:
I factored them out, it's still not super neat but seems to be a bit better than before.